app/code/Magento/Config/App/Config/Source/RuntimeConfigSource.php

```diff
@@ -54,13 +54,13 @@ public function __construct(
      * Get initial data.
      *
      * @param string $path Format is scope type and scope code separated by slash: e.g. "type/code"
-     * @return array
+     * @return mixed
      * @since 100.1.2
      */
     public function get($path = '')
     {
         $data = new DataObject($this->loadConfig());
-        return $data->getData($path) ?: [];
+        return !is_null($data->getData($path)) ? $data->getData($path): [];
```

Comment:
```suggestion
        return $data->getData($path) === null ? null : $data->getData($path);
```

Reply:
thank you for your review 
also there is  one possibility that config value is actually set `NULL` then in that case result will be error message like `Configuration for path: "test/active/enable" doesn't exist` which will not be correct